docs/upgrade.md

```diff
@@ -7,8 +7,8 @@ Here are the typical steps to upgrade a repo based on `ProjectScaffold` to use `
 
 1. Run
 
-       dotnet tool install FSharp.Formatting.CommandTool --local
-       git add dotnet-tools.json   
+       dotnet new tool
```

Comment:
I think the template you want is `tool-manifest`

Reply:
Hmm, you're probably right. That being said `tool` has always worked for me, though I'm not sure why, it doesn't seem to be listed in the available manifests.